sys-apps/kmod/kmod-9999.ebuild

```diff
@@ -41,73 +41,33 @@ RDEPEND="
 "
 DEPEND="${RDEPEND}"
 BDEPEND="
-	doc? (
-		dev-util/gtk-doc
-		dev-build/gtk-doc-am
-	)
+	app-text/scdoc
+	doc? ( dev-util/gtk-doc )
 	lzma? ( virtual/pkgconfig )
 	zlib? ( virtual/pkgconfig )
 "
-if [[ ${PV} == 9999* ]]; then
-	BDEPEND+=" app-text/scdoc"
-fi
-
-src_prepare() {
-	default
-
-	if [[ ! -e configure ]] || use doc ; then
-		if use doc; then
-			cp "${BROOT}"/usr/share/aclocal/gtk-doc.m4 m4 || die
-			gtkdocize --copy --docdir libkmod/docs || die
-		else
-			touch libkmod/docs/gtk-doc.make
-		fi
-		eautoreconf
-	else
-		elibtoolize
-	fi
-
-	# Restore possibility of running --enable-static, bug #472608
-	sed -i \
-		-e '/--enable-static is not supported by kmod/s:as_fn_error:echo:' \
-		configure || die
-}
-
 src_configure() {
 	# TODO: >=33 enables decompressing without libraries being built in
 	# as kmod defers to the kernel. How should the ebuild be adapted?
-	local myeconfargs=(
-		--bindir="${EPREFIX}/bin"
-		--sbindir="${EPREFIX}/sbin"
-		--enable-shared
-		--with-bashcompletiondir="$(get_bashcompdir)"
-		$(use_enable debug)
-		$(usev doc '--enable-gtk-doc')
-		$(use_enable static-libs static)
-		$(use_enable tools)
-		$(use_with lzma xz)
-		$(use_with pkcs7 openssl)
-		$(use_with zlib)
-		$(use_with zstd)
+	local emesonargs=(
+		--prefix "${EPREFIX}"
+		-Dbashcompletiondir="$(get_bashcompdir)"
+		-Dfishcompletiondir="$(get_fishcompdir)"
+		-Dzshcompletiondir="$(get_zshcompdir)"
+		$(meson_use debug debug-messages)
+		$(meson_use docs manpages)
```

Comment:
This seems wrong for 2 reasons:

1. The use flag is named "doc", not "docs".
2. It should not control the manpages.

Reply:
Silly copy/paste mistake - thanks it's fixed now.